src/services/expenses.service.js

```diff
@@ -0,0 +1,54 @@
+const { Op } = require('sequelize');
+const { Expense } = require('../models/models').models;
+
+class ExpensesService {
+  getAll({ userId, categories, from, to }) {
+    const where = {};
+
+    if (userId) {
+      where.userId = userId;
+    }
+
+    if (categories) {
+      if (Array.isArray(categories)) {
+        where.category = {
+          [Op.in]: categories,
+        };
+      } else {
+        where.category = categories;
+      }
+    }
+
+    if (from && to) {
+      where.spentAt = {
+        [Op.between]: [from, to],
+      };
+    }
```

Comment:
Consider adding validation to ensure `from` and `to` are valid dates and `from` is not after `to`.